src/main/java/mServer/tool/M3U8Utils.java

```diff
@@ -0,0 +1,92 @@
+package mServer.tool;
+
+import mServer.crawler.sender.newsearch.Qualities;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Arrays;
+import java.util.EnumMap;
+import java.util.HashMap;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.HashMap'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)